### DISCUSSION TITLE
Using the tool on Windows system

### DISCUSSION THREAD

**Discussion by TJbiz** [lunasec-io/lunasec#358] [Help/Support]
I'm trying to run this tool with the 'scan' command on a Windows system. It works great for one directory, but I'd like to run it recursively on my whole system (starting with the C:\ directory). How can I achieve this?

Also, the ANSI color formatting used by the tool doesn't work in a Windows command prompt, so the output looks a bit weird.

**Comment by OlivierMasit** [comment]
https://github.com/lunasec-io/lunasec/discussions/364

**Comment by rickhenderson** [comment]
I've noticed that under Windows the Analyze option doesn't seem to recognize the options when placed AFTER the command, as indicated in the help.
